tests/acceptance/standard/test_integrations.py

```diff
@@ -68,6 +68,29 @@ def test__if_delete_works(self, project):
             integration = project.integrations.get(integration_name)
             assert integration.active is False
 
+    def test__if_delete_without_create_works(self, project_for_function):
+        config_integrations_delete = f"""
+        projects_and_groups:
+          {project_for_function.path_with_namespace}:
+            integrations:
+              jira:
+                delete: true
+              slack:
+                delete: true
+        """
+
+        run_gitlabform(config_integrations_delete, project_for_function)
+
+        for integration_name in ["jira", "slack"]:
+            try:
+                project_for_function.integrations.get(integration_name)
+            except GitlabGetError as e:
+                assert (
+                    e.response_code == 404
+                ), f"Integration {integration_name} should not exist, but got a different error: {e}"
+            else:
+                assert False, f"Integration {integration_name} should not exist."
```

Comment:
Instead of catching `GitLabGetError` exception, couldn't we assert that 0 integrations exist for the project? Basically something like:

```py
project_integrations = project.integrations.list(get_all=True)
assert len(project_integrations) == 0
```

Please let me know if you think this misses some validation aspect.

Reply:
Good point, I've pushed this change.